### PR TITLE
Settings page - Theme and Number of Email

### PR DESCRIPTION
**Pull request by bivashpandey**
Now, users are able to select the theme and number of emails they want to display in sent items and inbox.

Screen Shot 2020-12-12 at 9 52 08 PM
Screen Shot 2020-12-12 at 9 51 31 PM

### CODE REVIEW

File: client/pages/_app.js [hunk 1]
```diff
@@ -29,8 +32,19 @@ axios.interceptors.response.use(
  * @param {*} pageProps
  */
 function MyApp({ Component, pageProps }) {
+  // Bivash Pandey (A00425523) - Start
+  const [settings, setSettings] = useState({});
```
Comment (just1ngray) [inline client/pages/_app.js]: Root _app.js is storing settings, you may as well pass it down!

File: client/pages/components/EmailList.js [hunk 2]
```diff
@@ -14,7 +15,7 @@ import defaults from "../../utils/defaults";
 import filterEmails from "../../utils/filterEmails";
 import Pagination from "./Pagination";
 
-const EMAILS_PER_PAGE = 20;
+let EMAILS_PER_PAGE = 20;
```
Comment (just1ngray) [inline client/pages/components/EmailList.js]: This can be passed down from _app.js

**Review by just1ngray — COMMENTED** [review]
I'm making a couple tiny changes according to my comments... feel free to revert

**Review by bivashpandey — COMMENTED** [review]
Looks good

**Review by just1ngray — APPROVED** [review]
Love it. Great work. **Have you considered making the CustomButtons theme-dependant as well?** I think you'd have to do the standard
```
const [colour, setColour] = useState("some default");
useEffect(() => grab from localstorage and setColour(that), []);
```
since it'd be a real pain to have to pass down the colour setting into every instance of `CustomButton`. This would be an excellent way to use Redux, but it's too late for that now 😜

**Comment by bivashpandey** [comment]
> Love it. Great work. **Have you considered making the CustomButtons theme-dependant as well?** I think you'd have to do the standard
> 
> ```
> const [colour, setColour] = useState("some default");
> useEffect(() => grab from localstorage and setColour(that), []);
> ```
> 
> since it'd be a real pain to have to pass down the colour setting into every instance of `CustomButton`. This would be an excellent way to use Redux, but it's too late for that now 😜

Thanks a great idea (Matching Button color and theme). Too late for this semester. Hopefully in any other semester 😜.

**Pull request merged**